webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/tasklist/template/TaskTemplate.java

```diff
@@ -8,9 +8,11 @@
 package io.camunda.webapps.schema.descriptors.tasklist.template;
 
 import io.camunda.webapps.schema.descriptors.backup.Prio3Backup;
+import io.camunda.webapps.schema.descriptors.operate.ProcessInstanceDependant;
 import io.camunda.webapps.schema.descriptors.tasklist.TasklistTemplateDescriptor;
 
-public class TaskTemplate extends TasklistTemplateDescriptor implements Prio3Backup {
+public class TaskTemplate extends TasklistTemplateDescriptor
+    implements Prio3Backup, ProcessInstanceDependant {
 
```

Comment:
In the tasklist archiver we add only 3 ProcessInstanceDependents: `TaskTemplate`, `TaskTemplate` and `VariableIndex`, in the camunda exporter now we have 11, we do have to probably use later a different ProcessInstanceDependents for the tasklist archiver jobs, correct?
<img width="1076" alt="image" src="https://github.com/user-attachments/assets/9fa9a0ce-0a39-45dd-8799-a437a617eb4c">

Reply:
We won't need to do anything for Tasklist since we now merged the indices. So all we need is the `ProcessInstancesArchiverJob`. This was confirmed here: https://camunda.slack.com/archives/C06F0GLJNFM/p1730121413008019